src/pe_reports/__main__.py

```diff
@@ -0,0 +1,6 @@
+"""Code to run if this package is used as a Python module."""
+
+# Third-Party Libraries
+from report_generator import main  # noqa: F401
```

Comment:
This needs to be a relative import. The `noqa` should not be needed because you use this import in this file.

```suggestion
from .report_generator import main
```

Reply:
Resolved in 44ee65c